### DISCUSSION TITLE
Hydration error caused by typing in input

### DISCUSSION THREAD

**Discussion by ZdravkoHr** [vercel/next.js#47489] [Help] [answered]
### Summary

Hello! I noticed that if the user starts typing in an input right after the page loaded, a hydration error will be thrown. I assume that this happens because by the time of hydrating the client content (changed by the user) will not match the server content. What is the right approach? Should I just suppress the hydration warning or is there anything else that can be done?

**Comment by icyJoseph** [comment, marked as answer]
I think in the app dir this happens when hydration is interrupted because user interaction was detected.

Like just loading a page and clicking rapidly as it loads, can trigger this, last time I checked anyway... Not sure what to do, but I think this will/should be fixed upstream by React.

**Reply by ZdravkoHr** [reply]
Yes, it really happens on any interaction, not only  typing in input. I am using Redux in my app and what I noticed is that when I comment out the provider which wraps my page, the problem is gone. I thought it was something with redux, but then I commented out the redux code from the provider and the issue was still there.  Then I realized it was happening because the provider is a client component. Maybe that should be fixed by React as you said. Thanks for the answer!
